### PR TITLE
My first cat

### PR DESCRIPTION
**Pull request by esmehm**
Added hello kitty image
Added intro

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -8,16 +8,10 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.tsx</code> and save to reload.
+          I love cats. My favourite cat is Hello Kitty!
```
Comment (HannahBBell) [inline src/App.tsx]: I think India's cat would be offended

Comment (esmehm) [reply src/App.tsx]: Good point Hannah, thanks. I have updated it now

Comment (HannahBBell) [reply src/App.tsx]: Okay thanks

**Review by HannahBBell — APPROVED** [review]
All good

**Pull request merged**